src/logger.ts

```diff
@@ -0,0 +1,22 @@
+import pino, { Logger } from 'pino';
+import { ecsFormat } from '@elastic/ecs-pino-format';
+
+const logLevel = process.env.LOG_LEVEL || 'info';
+
+/*
+ * Logger using Pino and the Elastic Common Schema format to facilitate
+ * standardization across all log messaging. This format will allow us
+ * to more easily debug and track requests in OpenSearch.
+ */
+export const logger: Logger = pino({ level: logLevel,  ...ecsFormat })
```

Comment:
@briri Would we ever want a dynamic way to update the level in production, if we were trying to debug a production issue?

Reply:
Ok, thanks @briri. It is probably overkill. It was just a thought because I came across a situation before where I needed to change logging levels in production.